learn/app-development/widgets/chart/chart-widget.md

```diff
@@ -230,6 +230,24 @@ Page.chart4_1Beforerender = function(widget, chartInstance) {
 
 [![](/learn/assets/onbeforerender-code.png)](/learn/assets/onbeforerender-code.png)
 
+### Customizing charts with rounded corners
+Inorder to achieve rounded corners for column and bar charts add class **'rounded-cornered-chart'** and for area chart add class **'rounded-area-chart'** to the chart widget.
+
+#### Column Chart
+[![](/learn/assets/rounded-column-chart.png)](/learn/assets/rounded-column-chart.png)

```

Comment:
Can we fix the rounded edge so it is not so rounded?

Reply:
Sure, will fix it in upcoming releases and make changes in the pr accordingly.

Reply:
<img width="968" alt="Screenshot 2023-08-29 at 12 50 23 PM" src="https://github.com/wavemaker/docs/assets/1387046/922153b4-848a-4017-9c7b-03b18190c1ff">

<img width="948" alt="Screenshot 2023-08-29 at 12 50 05 PM" src="https://github.com/wavemaker/docs/assets/1387046/8c4a5e5a-c971-41d7-a345-576fa8f6c0d1">

Reply:
We can adjust the property values in css and can achieve these as well